api/models/linux/interfaces/interface.proto

```diff
@@ -48,3 +50,7 @@ message VethLink {
 message TapLink {
     string vpp_tap_if_name = 1;     /* Logical name of the VPP TAP interface (mandatory for TAP_TO_VPP) */
 };
+
+message LoopLink {
+
```

Comment:
In vpp/ifplugin, we didn't define empty "link" for interfaces that do not have any type-specific attributes: https://github.com/ligato/vpp-agent/blob/master/api/models/vpp/interfaces/interface.proto#L60-L68 (for example SOFTWARE_LOOPBACK ). I think it is less confusing for users, but it is up to the discussion which is better.

Reply:
If there are no plans to add anything it's probably better to avoid it. Less unnecessary work.